scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -59,18 +59,32 @@ RUN set -ex; \
 		tabulate \
 		z3-solver;
 
+# TODO: we could eliminate duplication by using a Dockerfile extension like:
```

Comment:
Moving `validate_checksum` to a common script would also be nice :)

Reply:
Yes, but I'm not sure if we should address this in the current PR. I just realized I should have removed this TODO. I originally included it to remind myself of alternatives while the PR was still a draft. Should I remove it now and create an issue instead, so we can refactor it later?

Reply:
No no, this is definitely not for this PR.